Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM openjdk:22-slim
+
+RUN \
+    # Update
+    apt-get update &&\
+    \
+    # APKTool
+    apt-get install --no-install-recommends -y curl zipalign &&\
+    curl -o /usr/local/bin/apktool https://raw.githubusercontent.com/iBotPeaches/Apktool/master/scripts/linux/apktool &&\
+    curl -L -o /usr/local/bin/apktool.jar $(curl -s https://api.github.com/repos/iBotPeaches/Apktool/releases/latest |grep browser_download_url |awk '{print $2}' |sed 's/"//g') &&\
+    chmod +x /usr/local/bin/apktool /usr/local/bin/apktool.jar &&\
+    \
+    # Android SDK
+    apt-get install -y --no-install-recommends \
+        git \
+        git-lfs \
```

Comment:
is `git-lfs` just coverage in-case its needed? I don't think anything Apktool requires it.

Reply:
It's recommended for Android SDK installation, but I would guess since this is installing only for support tools, I can remove it. I'll give it a test.

Reply:
Worked just fine.